Yuan transition guide restructure (#11778)
openvinotoolkit/openvino: docs/OV_Runtime_UG/migration_ov_2_0/preprocessing.md

# Preprocessing {#openvino_2_0_preprocessing}

> This guide introduces how preprocessing works in API 2.0 by comparing with preprocessing in the previous Inference Engine API, and demostrates how to migrate preprocessing scenarios from Inference Engine to API 2.0 via some code samples.

Reviewer: ```suggestion
This guide introduces how preprocessing works in API 2.0 by a comparison with preprocessing in the previous Inference Engine API. It also demonstrates how to migrate preprocessing scenarios from Inference Engine to API 2.0 via code samples.
```
